bin/ovn-k8s-overlay

```diff
@@ -230,14 +234,14 @@ def master_init(args):
               "addresses=" + "\"" + router_mac + "\"")
 
     create_management_port(node_name, args.master_switch_subnet,
-                           args.cluster_ip_subnet)
+                           args.cluster_ip_subnet, args.service_ip_subnet)
 
 
 def minion_init(args):
     fetch_ovn_nb()
 
     if not args.node_name or not args.minion_switch_subnet \
-       or not args.cluster_ip_subnet:
+       or not args.cluster_ip_subnet or not args.service_ip_subnet:
```

Comment:
Perhaps it might be sufficient to add the route for kubernetes service during master_init only?